modules/update-mco-process.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="mco-update-process_{context}"]
+= Machine Config Operator node updates
+The Machine Config Operator (MCO) applies a new machine configuration to each control plane and compute node. During this process, the MCO performs the following sequential actions in the cluster:
+
+. Cordon and drain all of the nodes
+
+. Update the operating system (OS) of each node
+
+. Reboot the nodes
+
+. Uncordon all of the nodes
+
+. Schedule workloads on the nodes
+
+[NOTE]
+====
+When a node is cordoned, workloads cannot be scheduled to it.
+====
+
+This process is repeated one pool of nodes at a time until each control plane and compute node is updated. The size of each pool is dependent on the value of `.spec.maxUnavailable`, which has a default value of one node.
```

Comment:
@evakhoni and @petr-muller here is the new content I wrote up to describe node pools. Feel free to leave any feedback. 

Specifically, perhaps we should also mention where this `.spec.maxUnavailable` configuration is located.